src/Native/Runtime/StackFrameIterator.cpp

```diff
@@ -249,6 +249,10 @@ void StackFrameIterator::InternalInit(Thread * pThreadToWalk, PTR_PInvokeTransit
         m_HijackedReturnValueKind = GCRK_Byref;
     }
 
+#elif defined(_TARGET_WASM_)
+    UNREFERENCED_PARAMETER(pPreservedRegsCursor);
```

Comment:
These changes should not be needed once we switch to portable flavor of the runtime

Reply:
Removed this one, but the rest of the changes in this file aren't under portable ifdefs so I had to keep them.